.github/workflows/stale-pr.yml

```
@@ -0,0 +1,19 @@
+name: "Close stale pull requests"
+on:
+  schedule:
+    - cron: "12 31 * * *" # arbitrary time not to DDOS GitHub
```

```suggestion
    - cron: "23 * * * *" # https://crontab.guru/#23_*_*_*_*
```
Let's run every hour instead of every day.